app/lib/actions/pulp/repository/presenters/ostree_presenter.rb

```diff
@@ -0,0 +1,78 @@
+module Actions
+  module Pulp
+    module Repository
+      module Presenters
+        class OSTreePresenter < AbstractSyncPresenter
```

Comment:
Have a feeling we are better off calling this OstreePresenter. I think foreman tasks has trouble picking this because of the mixed case. Check out how its used in https://github.com/Katello/katello/blob/ostree/app/models/katello/ostree_branch.rb#L2